### DISCUSSION TITLE
Quailty of life improvement ideas.

### DISCUSSION THREAD

**Discussion by NigelsNutInLaw** [Avnsx/fansly#82] [Ideas]
Firsty i want to say realy cool app. I have zero codeing knowlage and have know idea how difficult these ideas would be to implemet. I just thought I would suggest them

1. The abilty to ingore feed or message content. idea for config "download_page = feed, messages or both" or maybe prioritize one over the other.
(Reason being when i start scraping it starts with messages but i already have all that content and it can take a while to move to the feed)

2.  The abilty to select the quailty of a video download. One person i sub to uploads in 4k and that takes ages to download.

3. Very silmar to number 1 the abilty to choose between Photos and Videos.  idea for config "content_type = videos, photos or both"

I have looked through the [config options](https://github.com/Avnsx/fansly/wiki/Explanation-of-provided-programs-&-their-functionality#4-configini) and cannot see a way of doing any of this but if I am worng please let me know.

Thanks

**Comment by Avnsx** [comment]
Star the repo and I'll think about it

**Comment by NigelsNutInLaw** [comment]
I have stared it
________________________________
